### PR TITLE
Add full fetch to conditions

### LINKED ISSUE #690: Full fetch thegraph for conditions
Related to #656, #453

The changes to show more than 1000 lines in the list was implemented for Positions
 It would be nice to implement such functionality for conditions

### PR DESCRIPTION
**Pull request by hpmaxi**
Closes #690

I've removed some unused code, there is a similar situation for positions and hook `usePositions` but is still used in `ConditionDetails` and it is possible that we should need to modify the query of `usePositionList` so I left that code as it is.  Maybe in another round of code-elimination or never.

### CODE REVIEW

File: src/hooks/useConditionsList.tsx [hunk 1]
```diff
@@ -25,35 +28,54 @@ export const useConditionsList = (advancedFilter: AdvancedFilterConditions) => {
   } = advancedFilter
 
   const query = buildQueryConditionsList(advancedFilter)
-  // eslint-disable-next-line @typescript-eslint/no-explicit-any
-  const variables: { [k: string]: any } = {}
-  if (ReporterOracle.type === OracleFilterOptions.Custom)
-    variables['oracleNotIn'] = ReporterOracle.value
```
Comment (hpmaxi) [inline src/hooks/useConditionsList.tsx:31]: wrapped in useMemo

File: src/queries/CTEConditions.tsx [hunk 2]
```diff
@@ -130,46 +130,13 @@ export const buildQueryConditionsList = (
...
-    oracleNotIn && oracleNotIn.length > 0 ? '$oracleNotIn: [String]' : '',
-  ]
-    .filter((s) => s.length)
-    .join(',')
-
-  const variablesClause = variablesClauseInternal ? `(${variablesClauseInternal})` : ''
-
-  const query = gql`
-      query Conditions ${variablesClause} {
-        conditions(first: 1000 ${whereClause} , orderBy: createTimestamp, orderDirection: desc) {
```
Comment (hpmaxi) [inline src/queries/CTEConditions.tsx:172]: This was used only by a hook that wasn't used.

**Comment by gnosis-info** [comment]
Travis automatic deployment:
 https://pr710--conditionaltokensexplorer.review.gnosisdev.com

**Comment by gnosis-info** [comment]
Travis automatic deployment:
 https://pr710--conditionaltokensexplorer.review.gnosisdev.com

**Review by mariano-aguero — APPROVED** [review]

**Pull request merged**